views/scss/qti/_tooltip.scss

```diff
@@ -25,4 +25,11 @@
 
 .qti-item .tooltip{
     padding: 10px;
+}
+
+.tooltip-active {
+    position: static;
```

Comment:
Caution, this is introducing a side effect, the tooltip will be always placed at the same position in the page, and won't be displayed close to the related target.

<img width="694" alt="image" src="https://user-images.githubusercontent.com/1500098/52650885-6351ac80-2eeb-11e9-82be-4e364cc64fb6.png">